frontend/views/containers/notifications/NotificationList.vue

```diff
@@ -1,136 +1,95 @@
-    },
     ephemeral: {
       isLoading: false
     }
   }),
   components: {
-    Avatar
-  },
-  mounted () {
-    // TODO this (All dumb logic for now)
-
-    // Simulate view with no notifications
-    if (this.notificationCount === 0) {
-      return null
-    }
-
-    // Simulate view with only 1 notification
-    if (this.notificationCount === 1) {
-      this.ephemeral.isLoading = true
-      setTimeout(() => {
-        // handpick a notification that is "MEMBER_ADDED"
-        this.config.notifications = {
-          3213: fakeDBWithNotifications[3213]
-        }
-        this.ephemeral.isLoading = false
-      }, 500)
-      return
-    }
-
-    // Load notifications from fake DB
-    this.ephemeral.isLoading = true
-    setTimeout(() => {
-      this.config.notifications = fakeDBWithNotifications
-      this.ephemeral.isLoading = false
-    }, 1500)
+    AvatarUser
   },
   computed: {
     ...mapGetters([
-      'notificationCount'
+      'newNotifications',
+      'newNotificationCount',
+      'notificationCount',
+      'olderNotifications'
     ]),
-    dateNow () {
-      // Hardcoded so the dummy layout makes sense
-      return 1590823007327
-    },
     notifications () {
-      if (!this.config.notifications) {
-        return {}
-      }
-
-      const list = {}
-      const listNew = []
-      const listOlder = []
-
-      for (const nId in this.config.notifications) {
-        const { timestamp, type, data } = this.config.notifications[nId]
-        const listToPush = this.dateNow - timestamp < HOURS_MILLIS * 2 ? listNew : listOlder
-        listToPush.push(templates[type]({ data, timestamp }))
-      }
-
-      if (listNew.length > 0) {
-        list.new = listNew
-      }
-
-      if (listOlder.length > 0) {
-        list.older = listOlder
+      return this.$store.state.notifications
+    },
+    notificationLists () {
+      const defaultCategory = 'OLDER'
+      const lists = [
+        { title: L('NEW'), items: this.newNotifications },
+        { title: L('OLDER'), items: this.olderNotifications }
+      ].filter(list => list.items.length > 0)
+
+      // If the only currently non-empty list has a category like 'OLDER' or 'UNREAD', then its title does not need to be displayed.
+      // See https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=4107%3A0
+      if (lists.length === 1 && lists[0].title === defaultCategory) {
+        lists[0].title = ''
       }
-
-      return list
+      return lists
     }
   },
   methods: {
-    title: (type) => {
-      return {
-        new: L('New'),
-        older: L('Older')
-      }[type]
+    ageTag (item: Object): number {
+      return timeSince(item.timestamp)
     },
-    iconBg: (level) => {
+    iconBg (level: string): string {
       return {
         info: 'has-background-primary has-text-primary',
         success: 'has-background-success has-text-success',
         danger: 'has-background-danger has-text-danger'
       }[level]
+    },
```

Comment:
That makes sense, however to be able to use computed values here we'd need something like a distinct Notification component, since these values differ per notification.

Reply:
Yeap. They are calling inside the loop. You are right. 😉